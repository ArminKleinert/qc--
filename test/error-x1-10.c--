
register bits8 c;

section "data" 
{
    x:
    bits32[0];

}
