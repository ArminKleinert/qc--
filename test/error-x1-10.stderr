Character 49-59 Error: not a positive size
